app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_person
+    print 'Enter (1) to create a student or (2) to create a teacher: '
+    type = gets.chomp.to_i
+    case type
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Please enter a valid input'
+    end
+  end
+
+  def create_student
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp.to_i
+    puts 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp
+    person = Student.new(age, parent_permission, name: name)
+    @people.push(person)
+    puts 'Student successfully created'
+  end
+
```

Comment:
- [ ] Great job working on the functionality so far but when I try to create a new student, I face this error with the wrong number of arguments (given 2, expected 1), so kindly check on this error and try to solve the problem to let the user be able to create a student ⭐🔥💯🙏🏻

- When I try to create a student  ⬇
![image](https://github.com/H-Ramalan/school-library/assets/58553711/acf7c99b-27f7-4ce6-abd1-183edc090b60)